src/gigs-board/components/posts/List.jsx

```diff
@@ -49,7 +49,9 @@ function href(widgetName, linkProps) {
 }
 /* END_INCLUDE: "common.jsx" */
 
-console.log(props);
+initState({
+  period: "week",
+});
```

Comment:
I would just do 3 different `recency`: `hot-24h`, `hot-week`, `hot-month` instead of using inner state, and this way there will be a way to link to the specific hot page, currently, there is no way to link to "Hot 24h" page as "Hot" page uses "week" by default.

Reply:
Good point, I also implemented the version as you suggested. However, the hot is like a way of sorting and it works together with search. But search is rely on state so the link of `recency=hot-24h` would be confusing, if user inputs with search term and shares the link it will become the hot without search. Therefore let's not do so before the search term is also in query string.

Reply:
I see. Well, it goes wild already with all those combinations, and the lack of navigation API makes things even more confusing.